_locales/en/messages.json

```diff
@@ -2099,7 +2099,7 @@
 				"content": "</a>"
 			},
 			"link_start": {
-				"content": "<br><a target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://www.ghostery.com/products/plus?utm_source=gbe&utm_campaign=in_app_hstats_lm\">"
+				"content": "<br><a target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://www.ghostery.com/pricing?utm_source=gbe&utm_campaign=in_app_hstats_lm\">"
```

Comment:
Let's update this to use `GHOSTERY_BASE_URL` too, if you can figure out a way to do this that doesn't require too much reworking of the code. Maybe this link can be pulled out of the string and instead be implemented in the component as a `PanelToTabLink`?

Reply:
Good idea. Refactored to use `PanelToTabLink`!